db/db_impl/db_impl_files.cc

```diff
@@ -193,11 +193,14 @@ void DBImpl::FindObsoleteFiles(JobContext* job_context, bool force,
       }
     }
 
+    IOOptions io_opts;
+    io_opts.property_bag["skip_directories"] = "true";
```

Comment:
Is there a reason to use this in the "property_bag" rather than as a specific IOOption?

Reply:
Since this property will be used by only GetChildren API so adding to property_bag seemed more appropriate.